docs/data-product-studio/data-quality/failed-events/exploring-failed-events/file-storage/index.md

```diff
@@ -1,20 +1,28 @@
 ---
 title: "Accessing failed events in file storage"
-sidebar_label: "Using S3 or GCS"
-sidebar_position: 1
+sidebar_label: "In file storage"
+sidebar_position: 2
 ---
 
-When failed events are generated on your pipeline the raw event payload along with details about the failure are saved into file storage (S3 on AWS, GCS on Google Cloud).
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+On AWS and GCP, when failed events are generated on your pipeline, the raw event payload along with details about the failure are saved into file storage (S3 on AWS, GCS on Google Cloud).
 
 :::info Community Edition quick start guide on GCP
 
 If you followed the [Community Edition quick start guide](/docs/get-started/snowplow-community-edition/quick-start/index.md) on GCP, you will need to manually deploy the [GCS Loader](/docs/api-reference/loaders-storage-targets/google-cloud-storage-loader/index.md) to save failed events into GCS, as it’s currently not included in the Terraform scripts.
 
 :::
 
+## Retrieving raw data
+
 You can directly access and download examples of events that are failing from file storage, this is useful for further investigation and also required to design a recovery operation.
 
-## Retrieving raw data from S3 on AWS
+<Tabs groupId="cloud" queryString>
+  <TabItem value="aws" label="AWS" default>
 
 - Login to your AWS Console account and navigate to the sub-account that contains your Snowplow pipeline
```

Comment:
I'm confused by this section. Are the images below, about each of these bullet points? Is this list of bullets together forming Step 0, or is each one Step 1, Step 2 etc? I think the latter? In which case let's have the images interleaved into this bit.
Is the text underneath each image about the image above? or below?

Reply:
It’s just gibberish as it came from Wordpress as far as I can see. Fixed